packages/blade-old/src/atoms/Link/Link.native.js

```diff
@@ -55,6 +58,7 @@ Link.propTypes = {
 Link.defaultProps = {
   onClick: () => {},
   size: 'medium',
+  weight: 'bold',
```

Comment:
Shouldn't default be 'regular'?
```suggestion
  weight: 'regular',
```

Reply:
already made this change, but forgot to push it as husky was giving error 🙈 

pushing now!

Reply:
@chaitanyadeorukhkar made the suggested change, please review now.

Reply:
Update snapshots as well?

Reply:
done, sir! 🧪